module/Finna/src/Finna/RecordDataFormatter/Specs/Utils/RecordFieldsTrait.php

```diff
@@ -0,0 +1,1833 @@
+            'getAccessRestrictions',
+            'data-accrest.phtml',
+            [
+                'context' => [
+                    'class' => 'extendedAccess',
+                ],
+            ],
+        ],
+        'Access Restrictions' => [
+            'getAccessRestrictions',
+            'data-accrest.phtml',
+            [
+                'context' => [
+                    'class' => 'extendedAccess',
+                ],
+            ],
+        ],
+        'Accessibility Feature' => [
+            'getAccessibilityFeatures',
+            'data-escapeHtml.phtml',
+            [
+                'context' => [
+                    'class' => 'record-accessibility-features',
+                ],
+            ],
+        ],
+        'Accessibility Hazard' => [
+            'getAccessibilityHazards',
+            'data-escapeHtml.phtml',
+            [
+                'context' => [
+                    'class' => 'record-accessibility-hazard',
+                ],
+            ],
+        ],
+        'Actors' => [
+            'getPresenters',
+            'data-actors.phtml',
+            [
+                'context' => [
+                    'class' => 'recordPresenters',
+                ],
+            ],
+        ],
+        'Additional Information' => [
+            'getTitleStatement',
+            'data-addInfo.phtml',
+            [
+                'context' => [
+                    'class' => 'recordTitleStatement',
+                ],
+            ],
+        ],
+        'Additional Information AIPA' => [
+            'getAdditionalInformation',
+            'data-additionalInformation.phtml',
+            [
+                'context' => [
+                    'class' => 'recordAdditionalInformation',
+                    'title' => 'AdditionalInformation',
+                ],
+            ],
+        ],
+        'Additional Information Extended' => [
+            'getTitleStatementsExtended',
+            'data-addInfoExtended.phtml',
+            [
+                'context' => [
+                    'class' => 'recordTitleStatement',
+                    'title' => 'AdditionalInformation',
+                ],
+            ],
+        ],
+        'Age Limit' => [
+            'getAgeLimit',
+            'data-escapeHtml.phtml',
+            [
+                'context' => [
+                    'class' => 'recordAgeLimit',
+                ],
+            ],
+        ],
+        'Appraisal' => [
+            'getAppraisal',
+            'data-escapeHtml.phtml',
+            [
+                'context' => [
+                    'class' => 'recordAppraisal',
+                ],
+            ],
+        ],
+        'Archive' => [
+            'getParentArchives',
+            'data-hierarchyLinks.phtml',
+            [
+                'context' => [
+                    'class' => 'recordHierarchyLinks',
+                ],
+                'labelFunction' => [
+                    DefaultRecord::class,
```

Comment:
Olisko parempi näissä kaikissa
```suggestion
                    self::class,
```
Tosin tämä koko tarve staattisille metodeille kertoo, ettei tämä nyt ehkä oikein traittina ole parhaimmillaan, vaan voisi olla koko homma DefaultRecord:ssa sisällä.